src/screens/QuizzesMap/components/LevelButton.tsx

```diff
@@ -0,0 +1,76 @@
+import {StyleSheet, TouchableOpacity, View} from 'react-native';
+import React from 'react';
+import Txt from '../../../components/common/Txt';
+import LockIcon from '../../../components/icons/LockIcon';
+import {useNavigation} from '@react-navigation/native';
+
+type LevelButtonProps = {
+  level: number;
+  currentUserLevel: number;
+};
+
+const LevelButton = ({level, currentUserLevel}: LevelButtonProps) => {
```

Comment:
Here the levels buttons in quizzes map screen are all in the same column, It would be better if you can put some random placement for levels buttons, and try to draw some line (dashed line) between levels buttons to make it appear as a road between levels.